.github/workflows/gerrit-compose-required-tox-verify.yaml

```diff
@@ -108,11 +120,28 @@ jobs:
           repository: lfit/releng-reusable-workflows
           sparse-checkout: .github/actions/tox-run-action
           path: reusable-tox-run-action
+      - name: Pre-build path
+        if: ${{ inputs.PRE_BUILD_SCRIPT_PATH != '' }}
+        # yamllint disable-line rule:line-length
+        run: |
+          echo "PRE_BUILD=${{ inputs.PRE_BUILD_SCRIPT_PATH }}" >> "$GITHUB_ENV"
+      - name: Pre-build script
+        if: ${{ inputs.PRE_BUILD_SCRIPT != '' }}
+        run: |
+          echo "${{ inputs.PRE_BUILD_SCRIPT }}" > "${{ github.run_id }}.sh"
+          chmod +x "${{ github.run_id }}.sh"
```

Comment:
I think we can simplify this in a single "Determine pre-build script" step with an if-else run. But this code is still good.

Reply:
Yeah, I always try to do things in the way native to Actions, but this might be somewhere that it's worthwhile to just use a shell script.